client/my-sites/themes/sheet.jsx

```diff
@@ -45,19 +43,7 @@ export const ThemeSheet = React.createClass( {
 	},
 
 	onPrimaryClick() {
-		let action;
-
-		if ( ThemeHelpers.isPremium( this.props ) && ! this.props.purchased && this.props.isLoggedIn ) { //FIXME: purchased ENOENT
-			action = purchase( this.props, this.props.selectedSite, 'showcase-sheet' );
-		} else if ( this.props.active ) { //FIXME: active ENOENT
-			action = customize( this.props, this.props.selectedSite );
-		} else if ( this.props.isLoggedIn ) {
-			action = activate( this.props, this.props.selectedSite, 'showcase-sheet' );
-		} else {
-			action = signup( this.props );
-		}
-
-		this.props.dispatch( action );
+		this.props.dispatch( signup( this.props ) );
```

Comment:
It is there for future usage, but given that it doesn't work it is fine to rip it out for now.